hoverctl/file.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+func WriteFile(filePath string, data []byte) error {
+	basePath := path.Dir(filePath)
+	fileName := path.Base(filePath)
+	log.Debug(basePath)
+
+	err := os.MkdirAll(basePath, 0777)
```

Comment:
Minor one, but should we being using 777 for the directory (we're using 644 for the file below)?